doc/manpages/bob-query-meta.rst

```diff
@@ -0,0 +1,38 @@
+
+.. _manpage-query-meta:
+
+bob-query-meta
+==============
+
+.. only:: not man
+
+   Name
+   ----
+
+   bob-query-meta - Query meta information
+
+Synopsis
+--------
+
+::
+
+    bob query-meta [-h] [-D DEFINES] [-c CONFIGFILE] [-r] [-f FILTER]
+                   PACKAGE [PACKAGE ...]
+
+Description
+-----------
+
+This command lists meta variables from the metaEnvironment section of the recipe. You can
+filter the output using a regular expression.
+
+Options
+-------
+
+``-c CONFIGFILE``
+    Use config File
+
+``-D DEFINES``
+    Override default environment variable
+
+``-f FILTER``
+    Filter for metaVars matching a given regex.
```

Comment:
Do we need a regex here? A shell pattern should be sufficient and is easier to write.

Also, the implementation is not like a filter (that throws away matching variables) but more like "grep". I would keep the '-f' switch for the future to implement custom formats like the other query commands....